pages/linux/bully.md

```diff
@@ -0,0 +1,13 @@
+# bully
+
+> Used to bruteforce WPS pin of a wireless access point.
+> Necessary information must be gathered with `airmon-ng` and `airodump-ng` before using `bully`.
+> More information: <https://www.kali.org/tools/bully/>.
+
+- Display help information:
```

Comment:
```suggestion
- Display help:
```
This is what we use for help command across all our places. Also, can you move this command to last?

Reply:
Sure